<launch>
    <!-- LLM Server -->
    <include file="$(find tello_llm_ros)/launch/llm_service.launch" />

    <!-- Test Case -->
    <node name="llm_service_tester" pkg="tello_llm_ros" type="llm_test.py" output="screen">
        <param name="test_cases_file" value="$(find tello_llm_ros)/config/test_cases.json"/>
    </node>
</launch>